### ISSUE TITLE
AdminDisableUser does not revoke tokens

### ISSUE THREAD

**Issue by KristobalJunta** [jagregory/cognito-local#381]
1. [InitiateAuth](https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_InitiateAuth.html) API call with USER_PASSWORD_AUTH flow returns tokens, namely: AccessToken, IdToken, RefreshToken.

2. If [AdminDisableUser](https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_AdminDisableUser.html) is called after that - user tokens are invalidated. As per documentation:
> Deactivates a user and revokes all access tokens for the user. 

3. If a new InitiateAuth is attempted after that with REFRESH_TOKEN_AUTH flow and a token saved from p1 - real AWS Cognito returns an error, due to the token being revoked.

4. On the other hand, cognito-local does not revoke tokens after p2 and the second InitiateAuth completes successfully.

**Comment by kadyrleev** [comment]
Yep, userPasswordAuthFlow is currently missing a check for Enabled status.
